python/setup.py

```diff
@@ -71,7 +71,7 @@
     ext_modules=cythonize(extensions, language_level="3"),
     zip_safe=False,
     install_requires=["numpy", "pillow", "pyarrow>=10,<11", "requests", "pandas"],
-    extras_require={"test": ["pytest>=6.0", "duckdb>=0.6,<0.7", "click", "requests_mock", "hypothesis"]},
+    extras_require={"test": ["pytest>=6.0", "duckdb==0.6.0", "click", "requests_mock", "hypothesis"]},
```

Comment:
Pinned duckdb here because our S3 artifacts for duckdb extension is still on 0.6.0.